common/credential-storage/src/backends/memory.rs

```diff
@@ -14,7 +15,6 @@ pub struct CoconutCredentialManager {
 #[derive(Default)]
 struct CoconutCredentialManagerInner {
     credentials: Vec<StoredIssuedCredential>,
-    credential_usage: Vec<CredentialUsage>,
```

Comment:
what about freepasses though

Reply:
They behave like ticket book so we don't need that